src/InputFileResolver.ts

```diff
@@ -33,11 +38,26 @@ export default class InputFileResolver {
       if (_.isObject(maybeInputFileDescriptor)) {
         if (Object.keys(maybeInputFileDescriptor).indexOf('pattern') === -1) {
           throw Error(`File descriptor ${JSON.stringify(maybeInputFileDescriptor)} is missing mandatory property 'pattern'.`);
+        } else {
+          maybeInputFileDescriptor = maybeInputFileDescriptor as InputFileDescriptor;
+          if (isWebUrl(maybeInputFileDescriptor.pattern) && maybeInputFileDescriptor.mutated) {
+            throw new Error(`Cannot mutate web url "${maybeInputFileDescriptor.pattern}".`);
+          }
         }
       }
     });
   }
 
+  private validateMutationArray(mutationArray: Array<string>) {
+    if (mutationArray) {
+      mutationArray.forEach(mutation => {
+        if (isWebUrl(mutation)) {
+          throw new Error(`Cannot mutate web url "${mutation}".`)
```

Comment:
You're missing a semicolon here: https://www.bithound.io/github/stryker-mutator/stryker/blob/869ee89d6fec2bf3ea98d36491e4efc7aaec8658/src/InputFileResolver.ts